app/AppKernel.php

```diff
@@ -23,6 +23,7 @@ public function registerBundles()
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new CodeExplorerBundle\CodeExplorerBundle(),
             new AppBundle\AppBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
```

Comment:
The dev environment requires ``sensio/generator-bundle`` to be bootable, which is not a regular requirement and so is not available on heroku (as it runs composer with ``--no-dev``)